app/controllers/ratings_controller.rb

```diff
@@ -0,0 +1,24 @@
+class RatingsController < ApplicationController
```

Comment:
seems like you use nested routes, for this reason we should move this controller to `app/controllers/articles/ratings_controller.rb`
```
resources :articles, only: :show do
  resources :ratings, only: %i[create update]
end
```